packages/vue-client/src/components/GameCreation/SFractionalConfigForm.vue

```diff
@@ -45,30 +45,30 @@ function removeSecondaryColor(): void {
     <BoardConfigForm @config-changed="setBoardConfig($event)" />
     <label>Komi</label>
     <input type="number" step="0.5" v-model="config.komi" />
-    <div class="row">
-      <label class="grow">Secondary Colors</label>
-      <button
-        class="no-flex"
-        v-on:click="addSecondaryColor()"
-        :disabled="config.secondary_colors.length >= maxNrOfSecondaryColors"
-      >
-        +
-      </button>
-      <button
-        class="no-flex"
-        v-on:click="removeSecondaryColor()"
-        :disabled="config.secondary_colors.length <= 1"
-      >
-        -
-      </button>
-    </div>
+
+    <label>Secondary Colors</label>
+
     <input
       type="color"
       v-for="(_, index) in config.secondary_colors"
       :key="index"
       v-model="config.secondary_colors[index]"
     />
   </form>
+  <!-- Note: I would like to place these buttons above the color inputs rather than below. -->
```

Comment:
Possible fix: `type="button"`

https://stackoverflow.com/questions/7803814/how-can-i-prevent-refresh-of-page-when-button-inside-form-is-clicked